cmd/query/main.go

```diff
@@ -46,6 +49,14 @@ func parseopts() opts {
 	return opts
 }
 
+// @title        VDS-slice API
```

Comment:
I suspect we are just never changing the name, are we?..
OK... I will have to learn to live with it then, even though I really find it misleading